### DISCUSSION TITLE
How to avoid a CW culture clash?

### DISCUSSION THREAD

**Discussion by technige** [mastodon/mastodon#20237] [General]
Firstly, hello, and thank you all for this incredible software :)

With so many refugees coming in from Twitter, many of whom do not have a technical background, it is noticeable that there is some (still currently mild) tension around the usage of content warnings. The scenario I've noticed most often concerns the posts of legal and political experts - most of which is obviously legal/political in nature. With a community guideline in place to CW any political content, we are essentially saying to these people that we all need to be "warned" about everything they have to say. Flipping this on its head, how would I feel about having to tag all technology content with a warning? I might feel that this network is not really somewhere that I belong. Of course this feature was put in place for very good reasons. But can it hold up as-is while mainstream adoption is occurring?

Having given it some thought, and having observed what's going on right now, I think a large part of this may simply be a naming issue - specifically in the word "warning". We want people to be able to choose which content they see and which they don't by default. But if this feature were re-imagined as a subject line, or a list of tags/keywords, or something else, could we make it more inclusive for people from all backgrounds? Most people surely wouldn't mind tagging their content, and we could all potentially choose which topics we want to see by default, and which we want to hide.

Thanks for listening, what do you all think?

**Comment by Psykatte** [comment]
Adding to this, setting tags in the actual CW header and being able toggle CW tags to default to displaying in full on a per-account basis would be very useful. Instances could at that point default accounts to blocking certain tags based on local guidelines, and those tags wouldn't carry over to other instances with different CW culture. This would negate most need for a branching CW system and would allow CWs to be dynamic and easy to control.

The downfall to this idea is that other communities won't need to add CWs for certain things, so boosting it to a community that requires such warnings would be problematic. This would require ability to add CWs to boosts to prevent infringement upon local guidelines.

Here's an example of how simple I envision this would look:
![cw_overhaul](https://user-images.githubusercontent.com/82907907/200965407-d3e3cc74-2879-4503-b5ff-5d2f9642064a.png)

**Comment by ingsme** [comment]
I hear that most people have in their settings to expand CW automatically. I know I do. Pressing show more on every other post became old pretty fast.

Remove Content Warning (the name i terrible) and replace it with a mandatory subject field. People who then feel the need to shield themself can decide to open only posts that are interested in and the rest of us gain an additional content indicator.

**Comment by Zahlenzauberin** [comment]
Hallo, over the last few days I thought a lot about this problem. As most of the CW set at the moment are absolutely useless for me, so I have set them to be expanded by default at the moment. But there are some I'd really like to keep. So here is my suggestion make it possible to filter which CW are automatically expanded and which aren't by default.
Let's say white flowers are something I'd like to have behind a CW. I could set a filter "white flowers" and all posts with the CW "white flowers" would be kept folded in, while all other posts with a CW would be automatically expanded.

**Comment by DiggidyDave** [comment]
> Most people surely wouldn't mind tagging their content

This is true, but part of the problem is that it is not necessarily obvious to the author of a post when something should have a CW tag.  This may not be a problem within tight knit "subcultures" but at scale it definitely will be. The people who most appreciate the CWs, the communities who are most impacted by some particular topic, are more likely to realize that a CW would be helpful for a post.

Rather than relying entirely on authors to tag things, has there been any consideration on inverting the ownership over the tags, and allowing post-hoc, community/crowd-sourced tagging of posts?

**Comment by Sirablopp** [comment]
The topic about renaming content warnings is already discussed over here, with various suggestions for other names. :)
https://github.com/mastodon/mastodon/issues/20117

That said, from what I read the biggest concern seems to be about the very wide definition of the term "politics", which is all over social media and media media often also refered to everyday struggles with problems in society or even the very existence of all kinds of minorities. There are also journalists whose very work is tooting about what's new in the world and I've seen some toots with confusion about if they have to use a CN for basically everything they post.
Renaming content warnings can help to make clearer how they are being used on Mastodon by many people (not just as actual warning but also spoilers or what a person considers off-topic) and also help for better acceptance as tool. However, some people should also reconsider what exactly they ask of their users to hide and clarify their rules and what they mean with "politics" specifically. Do they mean articles about political changes? Ranting about statements from politicians? Do they want people to mark what country's politics are about on this international platform, so people from Indonesia aren't facing tons of "go vote" from the US? Or do they consider everything related to the life of a minority "politics"? Choosing the right instance for you requires you to know, what you have to expect and clarification on a broad term like this can help a lot. I don't think there is a technical solution to this though, apart from maybe reminding mods to be precise when they set these rules.

(And at times people might also reconsider what exactly they see as something that needs a content note/warning/subject. Although not affected in a big way myself, I can see how minorities who have been shunned for speaking up about their troubles or are being shut out from the public eye currently have the impression they'd have to hide their every-day struggles in their lives with CWs/CNs on Mastodon as well... Not saying this is how it is handled on all instances, it's the impression some new people got based on a few replies they had to face.)

**Reply by technige** [reply]
Thanks for providing the link to the other thread. Despite searching, I missed this. We should close the thread here to avoid duplication.

**Comment by CEbbinghaus** [comment]
Just an idea I had but would there be any value to creating default CW(CN) Tags? e.g Hover over the CW(CN) button and get a dropdown of [Politics, NSFW, Violence, Mental Health, Trauma, etc...] that people can just select instead of having to type out their own every time? this would also allow for filtering based specifically on those CW(CN) Tags to remove any content tagged with that specific tag. This could also be implemented in both a backward compatible way as well as having a "better" UI showing if it's tagged.
